docker/scripts/build-docker.sh

```diff
@@ -58,7 +58,6 @@ run_build() {
       break
     fi
   done
-  echo $DOCKER_SQUASH
```

Comment:
What's the reason for removing this?

Reply:
It looks like it is a debug flag here only, without meaning.
Most of the time (at least i never see other than this) is showing `False` and this is it.